src/Hydrators/ProjectHydrator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ProjectManagementApi\Hydrators;
+
+use DateTime;
+
+// these need to be deleted when the classes are created
+use PDO as DatabaseConnection;
+use ProjectManagementApi\Hydrators\ProjectHydratorDemo\Project;
+
+// uncomment the below when ready to replace the above
+// use ProjectManagementApi\DatabaseConnection; 
+// use ProjectManagementApi\Entities\Project; 
+
+class ProjectHydrator
+{
+    public static function getAllProjects(DatabaseConnection $db)
+    {
+        $query = $db->prepare('SELECT id, `name`, client_id, deadline FROM projects');
+        $query->execute();
```

Comment:
is prepare/execute being used (without binding)

Reply:
technically don't need prepare and execute, just ->query. But no functional difference.